modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSlimServerCodegen.java

```diff
@@ -99,6 +87,43 @@ public String getHelp() {
         return "Generates a PHP Slim Framework server library.";
     }
 
+    @Override
+    public void setInvokerPackage(String invokerPackage) {
+        super.setInvokerPackage(invokerPackage);
+        apiPackage = invokerPackage + "\\" + apiDirName;
+        modelPackage = invokerPackage + "\\" + modelDirName;
```

Comment:
```diff
+        apiPackage = invokerPackage + "\\" + apiDirName;
+        modelPackage = invokerPackage + "\\" + modelDirName;
```

As the method name `setInvokerPackage()` implies, it should set only `invokerPackage`. 💡

Reply:
@ackintosh 
I've copied it from PhpSymfony generator.
https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSymfonyServerCodegen.java#L481-L490

Where I should place these changes then?